operations/commit_queue.go

```diff
@@ -444,10 +529,16 @@ func (p *mergeParams) uploadMergePatch(conf *ClientSettings, ac *legacyClient) e
 	}
 	patchParams.Description = fmt.Sprintf("%s %s", commitQueuePatchLabel, commits)
 
-	patch, err := patchParams.createPatch(ac, conf, diffData)
+	if err = patchParams.validateSubmission(diffData); err != nil {
+		return err
+	}
+	patch, err := patchParams.createPatch(ac, diffData)
 	if err != nil {
 		return err
 	}
+	if err = patchParams.displayPatch(conf, patch); err != nil {
+		grip.Error("problem printing patch information")
```

Comment:
This prints to the user so it should read as a complete sentence.

Reply:
🙃 
done